libs/components/theme/src/lib/styles/themes/modern/_theme.scss

```diff
@@ -86,6 +86,6 @@
     }
   }
   .sky-rounded-corners {
-    border-radius: $sky-theme-modern-box-border-radius-default;
+    border-radius: var(--sky-border-radius-s);
```

Comment:
We use this class in toast - updated here to allow us to tokenize when we did this component.